specification/communication/data-plane/readme.go.md

```diff
@@ -0,0 +1,19 @@
+## Go
```

Comment:
If the go SDK generation for data plane is not needed, please just remove this file.